orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/CompleteStageHandler.kt

```diff
@@ -103,13 +86,12 @@ class CompleteStageHandler(
           log.error("Failed to construct after stages", e)
           stage.status = TERMINAL
           stage.endTime = clock.millis()
-          repository.storeStage(stage)
         }
 
         stage.includeExpressionEvaluationSummary()
         repository.storeStage(stage)
 
-        if (stage.status in listOf(SUCCEEDED, FAILED_CONTINUE)) {
+        if (stage.status in listOf(SUCCEEDED, FAILED_CONTINUE, SKIPPED)) {
```

Comment:
This only affects stages that have no synthetics _or_ tasks (of which there are zero, so it's theoretical). Still, it's not right they'd zombie.